### DISCUSSION TITLE
貌似只加载GOpoc，没有加载ymlpoc

### DISCUSSION THREAD

**Discussion by purple-WL** [hktalent/scan4all#29] [Q&A] [answered]
直接https://github.com/hktalent/scan4all/releases/download/2.6.1/scan4all_2.6.1_macOS_amd64.zip下载，求告知运行前需要准备什么环境吗，感觉没有加载poc，漏洞靶场都扫不出漏洞

**Comment by hktalent** [comment, marked as answer]
pocs_yml 检测前提是需要有指纹命中他们

**Reply by purple-WL** [reply]
谢谢，从https://github.com/hktalent/scan4all/releases/download/2.6.1/scan4all_2.6.1_macOS_amd64.zip直接下载的话直接运行就行，无需安装其他东西吧

**Reply by hktalent** [reply]
特别注意，xray POC检测需要有指纹命中后才会触发检测

**Reply by hktalent** [reply]
不需要安装其他东西了，所有poc会自动释放到当前目录 config 中
